resources/libpostal/dictionaries/nb/concatenated_suffixes_separable.txt

```diff
@@ -1,6 +1,5 @@
-allè|alle
```

Comment:
You should not change content in `libpostal` folder, please blacklist them in `pelias/dictionaries/libpostal` folder instead.

Reply:
Ahh, so _that's_ what that Pelias directory does...